flink-runtime/src/main/java/org/apache/flink/runtime/state/UserFacingMapState.java

```diff
@@ -34,8 +34,6 @@
 
 	private final MapState<K, V> originalState;
 
-	private final Map<K, V> emptyState = Collections.<K, V>emptyMap();
```

Comment:
`UserFacingMapState` already have the same semantics for these methods, I don't think we must refactor this class.